.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -88,16 +89,19 @@ body:
     attributes:
       label: Anything in the Host logs that might be useful for us?
       description: >
-        Supervisor Logs can be found under [Configuration panel -> Add-ons, Backup & Supervisor -> System](https://my.home-assistant.io/redirect/supervisor_logs/), then choose Log Provider `Host`.
+        Host Logs can be found in [Settings -> System -> Logs](https://my.home-assistant.io/redirect/logs/)
+        then choose `Host` in the top right.
       render: txt
   - type: textarea
     attributes:
       label: System Health information
```

Comment:
```suggestion
      label: System information
```

Reply:
Ok I'll do this but then we need to correct the my link. It's confusing that we ask for "System information" but have a button called "System Health". I was trying to split the difference here.